### PR TITLE
[TA] More renamings

### PR DESCRIPTION
**Pull request by mssfang**
ISO4217 -> iso4217
script -> scriptKind

### CODE REVIEW

File: sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/CurrencyResolutionPropertiesHelper.java [hunk 1]
```diff
@@ -14,7 +14,7 @@ private CurrencyResolutionPropertiesHelper() { }
      * Type defining the methods to set the non-public properties of an {@link CurrencyResolution} instance.
      */
     public interface CurrencyResolutionAccessor {
-        void setISO4217(CurrencyResolution currencyResolution, String iso4217);
+        void setIso4217(CurrencyResolution currencyResolution, String iso4217);
```
Comment (kristapratico) [inline sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/CurrencyResolutionPropertiesHelper.java:17]: shouldn't this rename come from regenerating on the swagger (once it's merged)?

**Review by kristapratico — APPROVED** [review]

File: sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/DetectedLanguagePropertiesHelper.java [hunk 2]
```diff
@@ -32,7 +32,7 @@ public static void setAccessor(
         accessor = detectedLanguageAccessor;
     }
 
-    public static void setScript(DetectedLanguage detectedLanguage, ScriptKind script) {
-        accessor.setScript(detectedLanguage, script);
+    public static void setScriptKind(DetectedLanguage detectedLanguage, ScriptKind scriptKind) {
```
Comment (joseharriaga) [inline sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/DetectedLanguagePropertiesHelper.java:35]: Out of curiosity: What was the feedback for renaming `Script` to `ScriptKind`? I do think `ScriptKind` sounds a little better, and I'm wondering if I need to make this change too.

Comment (mssfang) [reply sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/DetectedLanguagePropertiesHelper.java:35]: In Java, We usually match the return type in the method name unless it has a specific reason. I was trying to align language by using `getScript()` but Java architect suggests following the original naming pattern. Also, I also do feel `getScriptKind` sounds more reasonable.

File: sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/CurrencyResolutionPropertiesHelper.java (hunk 1 above)
Comment (mssfang) [reply sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/CurrencyResolutionPropertiesHelper.java:17]: Once the new changes are merged, the model in the implementation folder will be updated, but this model will still stay the same. Java has two CurrencyResolution models, one explored to the public (which is manually copied and edited which will map to the model in the implementation) and another one hide in the implementation folder (which is autogenerated.

**Pull request merged**